Corgibytes.Freshli.Cli/Functionality/Git.cs

```diff
@@ -100,69 +100,66 @@ private void Delete()
 
     private void Clone(string gitPath)
     {
-        var cloneProcess = new Process
-        {
-            StartInfo = new ProcessStartInfo()
-            {
-                FileName = gitPath,
-                WorkingDirectory = Directory.FullName,
-                Arguments = $"clone {Url} .",  // clone directly in the working directory
-                RedirectStandardOutput = true,
-                RedirectStandardError = true
-            }
-        };
-        cloneProcess.Start();
-        cloneProcess.WaitForExit();
-
-        if (cloneProcess.ExitCode != 0)
+        var stdErrBuffer = new StringBuilder();
+        var command = CliWrap.Cli.Wrap(gitPath).WithArguments(
+                args => args
+                    .Add("clone")
+                    .Add(Url)
+                    .Add('.')
+            )
+            .WithValidation(CommandResultValidation.None)
+            .WithWorkingDirectory(Directory.FullName)
+            .WithStandardErrorPipe(PipeTarget.ToStringBuilder(stdErrBuffer));
+
+        var task = command.ExecuteAsync().Task;
+        task.Wait();
```

Comment:
Calling `Result` on the next line implies `Wait()`, but I see little harm in being explicit here.

Reply:
It *wasn't* implying it, which is why I mentioned in Slack about the race condition.